cmd/up.go

```diff
@@ -19,5 +21,114 @@ func init() {
 }
 
 func up(cmd *cobra.Command, args []string) {
-	fmt.Println("TODO")
+
+	//read the harbor compose file
+	harborCompose := DeserializeHarborCompose(HarborComposeFile)
+
+	//read the docker compose file
+	dockerCompose := DeserializeDockerCompose(DockerComposeFile)
+	if dockerCompose.Version != "2" {
```

Comment:
should this check be in the Deserialize function? That way we don't have to do it in every function.